Cesium.IntegrationTests/complex_subscript_expressions.c

```diff
@@ -0,0 +1,7 @@
+typedef struct { int x[10]; } foo;
```

Comment:
It seems to be we have test for this https://github.com/ForNeVeR/Cesium/blob/main/Cesium.IntegrationTests/struct_subscription.ignore.c
If you can remove `.ignore` it will start running.

Reply:
Also I assume @ForNeVeR ask for new test somewhere in the [Cesium.CodeGen.Tests](https://github.com/ForNeVeR/Cesium/tree/main/Cesium.CodeGen.Tests) project